developer-docs-site/docs/tutorials/index.md

```diff
@@ -22,6 +24,22 @@ If running macOS, install the following packages in the order specified to take
     ```
 1. **Poetry**: Install Poetry from [https://python-poetry.org/docs/#installation](https://python-poetry.org/docs/#installation).
 
+## Adjust network and funding source
+
+For development purposes, the tutorials here assume you are working in the Aptos devnet network and therefore interacting with the devnet Aptos node and faucet service for creating and funding accounts. As noted in the relevant tutorials, these targets can be configured with the following environment variables:
+  * `APTOS_NODE_URL`
+  * `APTOS_FAUCET_URL`
+
+Here are the options for funding by network:
+  * *Testnet* - Use the faucet button in-wallet or use the testnet faucet at: https://aptoslabs.com/testnet-faucet.
```

Comment:
If we're mentioning in-wallet, we should probably tell them how to get a wallet / Petra?

Reply:
Linked.